script.js

```diff
@@ -54,7 +58,9 @@ const initLangSelector = () => {
 
     // Set lang selector
     const l = new URLSearchParams(window.location.search).get('l');
-    select.set(l ? decodeURIComponent(l) : shorten('Plain Text'));
+    fileExtToLang(l)
+        .then((lang) => select.set(decodeURIComponent(lang)))
+        .catch(select.set(shorten('Plain Text')));
```

Comment:
What about this:
- check for the `?l` query parameter
- if not found, check for a `?e` that can contain the file extension
- if not found, set to Plain Text

Reply:
Thanks for your help. PR updated.